pkg/loopback/attach_loopback.go

```diff
@@ -50,29 +52,40 @@ func openNextAvailableLoopback(index int, sparseName string, sparseFile *os.File
 
 	// Start looking for a free /dev/loop
 	for {
```

Comment:
Shouldn’t there be a limit against infinite loops? It can be a ridiculous number — 100 attempts, 1000 attempts — but infinitely looping (e.g. if some weird security software prevented usage of `/dev/loop*`), with no output even, would be bad.

Reply:
Probably yes; looks like systemd does 64 attempts with a random sleep delay.

Reply:
I've added an upper bound with 1000 attempts